src/uct/ib/dc/dc_mlx5.c

```diff
@@ -351,7 +352,7 @@ static ucs_status_t uct_dc_mlx5_iface_create_dci(uct_dc_mlx5_iface_t *iface,
         attr.full_handshake              = full_handshake;
         attr.rdma_wr_disabled            = (iface->flags & UCT_DC_MLX5_IFACE_FLAG_DISABLE_PUT) &&
                                            (md->flags & UCT_IB_MLX5_MD_FLAG_NO_RDMA_WR_OPTIMIZED);
-        attr.log_num_dci_stream_channels = iface->tx.log_num_dci_stream_channels;
+        attr.log_num_dci_stream_channels = iface->tx.num_dci_channels;
```

Comment:
should probably use ilog2 here
since left side has "log" and right side does not have "log" in the name

Reply:
> should probably use ilog2 here
> since left side has "log" and right side does not have "log" in the name

hmmm, `iface->tx.num_dci_channels` stores log-value of channels number... ok, it will store `real` value